src/altProgramAssociation/altProgramAssociation.controller.ts

```diff
@@ -279,11 +279,27 @@ export class ALTProgramAssociationController {
       timeframe,
     });
   }
+
+  // Class Teacher API
+
+  @Post("classTeacher")
+  @ApiBasicAuth("access-token")
+  @ApiOkResponse({ description: "ALT Rules" })
+  @ApiForbiddenResponse({ description: "Forbidden" })
+  async classTeacher(@Body() body: any, @Req() request: Request) {
+    const { filters } = body;
+
+    return this.altProgramAssociationService.classTeacher(request, {
```

Comment:
@suraj-tekdi as we are removing the controller we should remove the controller as well